services/app-web/src/components/Form/FieldYesNo/FieldYesNo.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { useField } from 'formik'
+import { Fieldset } from '@trussworks/react-uswds'
+import { FieldRadio } from '../FieldRadio/FieldRadio'
+import { PoliteErrorMessage } from '../../'
+
+import styles from './FieldYesNo.module.scss'
+
+/**
+ * This component renders a pair of radio buttons with Yes and No as the options.
+ *
+ * It relies on the Formik useField hook to work, so it must ALWAYS be rendered
+ * inside of a Formik form context.
+ *
+ * If you want to use these components outside a Formik form, you can use the
+ * ReactUSWDS components directly.
+ */
+
+export type FieldYesNoProps = {
+    name: string
+    label: string
+    showError: boolean
+    id: string
+} & JSX.IntrinsicElements['input']
+
+export const FieldYesNo = ({
+    name,
+    label,
```

Comment:
We should be using the label prop in contract details. It will solve a screenreader issue where when you go through the list the radios are announce with their values but the label text is not read.  We need to pass the label into the FieldYesNo > FieldRadio to address this bug I believe